tsdb/block.go

```diff
@@ -40,7 +40,7 @@ import (
 // IndexWriter serializes the index for a block of series data.
 // The methods must be called in the order they are specified in.
 type IndexWriter interface {
-	// AddSymbols registers all string symbols that are encountered in series
+	// AddSymbol registers all string symbols that are encountered in series
```

Comment:
The method signature used to be `AddSymbols(sym map[string]struct{})`, so I'm afraid we also need to update the content of the doc comment, not just the first word. It appears the current version of the method only adds a single symbol.

Reply:
OK,got,then I'll revise it and submit it again.Thx.

Reply:
It has been revised again. Thank you for your suggestion,please review again. @beorn7